recipe/meta.yaml

```diff
@@ -8,8 +9,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/dmlc/xgboost/releases/download/v{{ version }}/xgboost-{{ version }}.tar.gz
-  sha256: b6ce5870d03cc1233cad5ff8460f670a2aff78625adfb578c0b9eec3b8b88406
+  url: https://github.com/dmlc/xgboost/releases/download/v{{ version }}/xgboost-src-{{ version }}.tar.gz
+  sha256: 431222b47085b9c3504d77ef59cfa23ae4fe9d701085313f47217e49e8823326
```

Comment:
Adding `-src-` infix to the URL to match a similar change upstream: https://github.com/dmlc/xgboost/pull/11055

Should stay the same going forward